.gitignore

```diff
@@ -88,12 +88,16 @@ qrc_*.cpp
 # Qt creator
 *.pro.user
 
+# NetBeans
```

Comment:
In closed-source environments in which everyone uses the same IDE that is common. In open source software, where everyone uses their own editors/IDE/tools, it is less common. Only you know what files your editor produces and this may change from version to version.
The canonical way to do this is thus to create your local gitignore. Add this to `~/.gitconfig`:
```
[core]
	excludesfile = /home/.../.gitignore_global
```
Then put your favourite tool's excrement filenames in that file :-)
Another option is to use per-project `.git/info/exclude`. These are not committed either.